culinary-canvas/capstone-project/backend/index.js

```diff
@@ -57,6 +88,187 @@ const storage = new CloudinaryStorage({
 });
 const upload = multer({ storage: storage })
 
+//find the lowest Nutrient
+const findLowestNutrient = (nutrientTotals) => {
+    let lowestNutrient = null;
+    let lowestValue = Infinity;
+
+    Object.keys(nutrientTotals).forEach((key) => {
+        if(nutrientTotals[key] < lowestValue){
+            lowestNutrient = key;
+            lowestValue = nutrientTotals[key]
+        }
+    });
+    return lowestNutrient;
```

Comment:
there should be a function for this. Maybe the reduce function here?: https://stackoverflow.com/questions/35527341/getting-all-minimal-values-from-dictionary-javascript